fabfile/setup.py

```diff
@@ -125,7 +125,12 @@ def copy_pem_file(user=None, host=None, key_name=None, environment=None):
         else:
             env.user = user
     if host is None:
-        env.host_string = environment.get('app_host_ip')
+        try:
+            env.host_string = environment.get('app_host_ip')
+        except:
```

Comment:
is there a certain exception we're looking for here? might be better than a blank exception